test_runner/performance/test_unlogged.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+import threading
+from typing import TYPE_CHECKING
+
+import pytest
+
+if TYPE_CHECKING:
+    from fixtures.neon_fixtures import NeonEnvBuilder
+
+
+@pytest.mark.timeout(10000)
+def test_unlogged(neon_env_builder: NeonEnvBuilder):
```

Comment:
Please add a comment explaining what this measures.

Reply:
Done